### PR TITLE
Added Quota documentation for Cluster Admins.

### PR DESCRIPTION
**Pull request by erictune**
Also documented Admission Control Plugins
for developers.

### CODE REVIEW

File: docs/resource_quota_admin.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+cpu                     0m      20
+memory                  0       1Gi
+pods                    5       10
+replicationcontrollers  5       20
+resourcequotas          1       1
+services                3       5
+```
+
+## Quota and Cluster Capacity
+Resource Quota objects are independent of the Cluster Capacity.  There are
```
Comment (derekwaynecarr) [inline docs/resource_quota_admin.md]: They are...

File: docs/resource_quota_admin.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+expressed in absolute units.  
+
+Sometimes more complex policies may be desired, such as:
+  - proportionally divide total cluster resources among several teams.
+  - allow each tenant to grow resource usage as needed, but have a generous
+    limit to prevent accidental resource exhaustion.
+
+Such policies could be implemented using ResourceQuota as a building-block, by
+writing a controller which watches every the quota usage and adjusts the quota
```
Comment (derekwaynecarr) [inline docs/resource_quota_admin.md]: Watches the quota usage....

**Comment by derekwaynecarr** [comment]
Thanks for putting this together.  Saved me a todo this week. Few minor nits

**Comment by erictune** [comment]
addressed comments

**Comment by derekwaynecarr** [comment]
LGTM

**Pull request merged**